embedchain/__init__.py

```diff
@@ -5,7 +5,7 @@
 from embedchain.apps.App import App  # noqa: F401
 from embedchain.apps.CustomApp import CustomApp  # noqa: F401
 from embedchain.apps.Llama2App import Llama2App  # noqa: F401
-from embedchain.apps.OpenSourceApp import OpenSourceApp  # noqa: F401
+from embedchain.apps.Open_Source_App import OpenSourceApp  # noqa: F401
```

Comment:
Can you please make the letters to be lowercase?

Reply:
sure

Reply:
done